lib/_http_outgoing.js

```diff
@@ -930,11 +931,8 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
 
   let ret;
   if (msg.chunkedEncoding && chunk.length !== 0) {
-    len ??= typeof chunk === 'string' ? Buffer.byteLength(chunk, encoding) : chunk.byteLength;
-    msg._send(NumberPrototypeToString(len, 16), 'latin1', null);
-    msg._send(crlf_buf, null, null);
-    msg._send(chunk, encoding, null, len);
-    ret = msg._send(crlf_buf, null, callback);
+    chunkBuffer = Buffer.concat([chunkBuffer, Buffer.from(chunk)]);
```

Comment:
Concatenating buffers is very slow. They require a memory allocation + memory copy. Just keep an `Array` of buffers.